store/store.go

```diff
@@ -52,6 +55,18 @@ func (cfg *Config) Check() error {
 		return fmt.Errorf("redis password is set, but endpoint is not")
 	}
 
+	// NOTE: we take the MaxBlobLengthBytes from verify package as it is done in memstore
+	// 8Mb in bytes is 8_388_608
+	if cfg.WeaveVMConfig.Enabled && verify.MaxBlobLengthBytes > 8_388_608 {
```

Comment:
knit - let's add a constant var for `8_388_608`

Reply:
done